### DISCUSSION TITLE
Support for deno

### DISCUSSION THREAD

**Discussion by sobrborsen** [directus/directus#3363] [Feature Requests]
I am trying to use directus/sdk-js with deno. Since it is not present in https://deno.land/x I have tried with jspm.dev, unpkg.com and skypack.dev, but they all fail when I try to run my application in the browser after I have `deno bundle` my application. https://deno.land/x#info describes how to it. So I hope you will publish it in https://deno.land/x :-)

**Comment by rijkvanzanten** [comment]
I have no idea if we can, and how it works. If this is something you'd like to take on feel free to open a PR with any required changes 🙂

**Comment by nickrum** [comment]
I have never worked with deno, but I think this should work out of the box using unpkg once #3299 lands.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
